ui/src/app/collection/collections-list.component.ts

```diff
@@ -111,34 +113,35 @@ export class CollectionsListComponent extends Whitelabelled {
     // return (this.selectedCollections?.length ?? 0) > 0
   }
 
-  publishVisible(skill?: ApiCollectionSummary): boolean {
-    if (skill !== undefined) {
-      return skill.publishDate === undefined
+  publishVisible(collection?: ApiCollectionSummary): boolean {
+    if (collection !== undefined) {
+      return collection.publishDate === undefined && !this.authService.isDisabledByRoles("COLLECTION_PUBLISH")
     } else if ((this.selectedCollections?.length ?? 0) === 0) {
       return false
     } else {
-      const unpublishedSkill = this.selectedCollections?.find(s => s.publishDate === undefined)
-      return unpublishedSkill !== undefined
+      const unpubCollection = this.selectedCollections?.find(s => s.publishDate === undefined)
```

Comment:
Sorry to dwell on naming so much.  I'd spell out unpub here, otherwise, I'd read it as "unpublishCollection" a boolean, meaning that I'd need to take action if true.

Reply:
Done deal.